xla/service/gpu/fusions/mlir/elemental_hlo_to_mlir.cc

```diff
@@ -1180,20 +1180,13 @@ absl::StatusOr<SmallVector<Value, 1>> HloToMlir(
 }  // namespace
 
 bool IsHloOpSupported(const HloInstruction* instr,
```

Comment:
We can remove the compute capability here now (well, technically, we could have removed it before, but now we really can :-)).

Reply:
FYI I did this in #16085.